packages/contracts-bedrock/scripts/DeployOwnership.s.sol

```diff
@@ -0,0 +1,164 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { console2 as console } from "forge-std/console2.sol";
+import { stdJson } from "forge-std/StdJson.sol";
+
+import { Safe } from "safe-contracts/Safe.sol";
+import { OwnerManager } from "safe-contracts/base/OwnerManager.sol";
+import { GuardManager } from "safe-contracts/base/GuardManager.sol";
+
+import { Deployer } from "scripts/Deployer.sol";
+
+import { LivenessGuard } from "src/Safe/LivenessGuard.sol";
+import { LivenessModule } from "src/Safe/LivenessModule.sol";
+
+import { Deploy } from "./Deploy.s.sol";
+
+struct SafeConfig {
+    uint256 threshold;
+    address[] owners;
+}
+
+struct SecurityCouncilConfig {
+    SafeConfig safeConfig;
+    uint256 livenessInterval;
+    uint256 thresholdPercentage;
+    uint256 minOwners;
+    address fallbackOwner;
+}
+
+/// @title Deploy
+/// @notice Script used to deploy and configure the Safe contracts which are used to manage the Superchain,
+///         as the ProxyAdminOwner and other roles in the system.
+contract DeployOwnership is Deploy {
```

Comment:
Feels like a bad abstraction that we have to inherit from this since `Deploy` is so big, but that's ok for now

Reply:
This should be able to inherit from `Artifacts` if it needs deploy config + reading in addresses

Reply:
We're using quite a few different utilities from `Deploy`, including the `broadcast` modifier, `_callViaSafe()`, `deploySafe()`, and in the next PR we're also using `deploySuperchainConfig()`.